docs/design/architecture.md

```diff
@@ -221,48 +490,35 @@ or `container`. Kata Containers will then use this annotation to decide if it ne
 respectively create a virtual machine or a container inside a virtual machine associated
 with a Kubernetes pod:
 
-```Go
-	containerType, err := ociSpec.ContainerType()
-	if err != nil {
-		return err
-	}
-
-	handleFactory(ctx, runtimeConfig)
-
-	disableOutput := noNeedForOutput(detach, ociSpec.Process.Terminal)
-
-	var process vc.Process
-	switch containerType {
-	case vc.PodSandbox:
-		process, err = createSandbox(ctx, ociSpec, runtimeConfig, containerID, bundlePath, console, disableOutput, systemdCgroup)
-		if err != nil {
-			return err
-		}
-	case vc.PodContainer:
-		process, err = createContainer(ctx, ociSpec, containerID, bundlePath, console, disableOutput)
-		if err != nil {
-			return err
-		}
-	}
-
-```
+| Annotation value | Kata VM created? | Kata container created? |
+|-|-|-|
+| `sandbox` | yes | yes (inside new VM) |
+| `container`| no | yes (in existing VM) |
 
 #### Mixing VM based and namespace based runtimes
 
 > **Note:** Since Kubernetes 1.12, the [`Kubernetes RuntimeClass`](https://kubernetes.io/docs/concepts/containers/runtime-class/)
 > has been supported and the user can specify runtime without the non-standardized annotations.
 
-With `RuntimeClass`, users can define Kata Containers as a `RuntimeClass` and then explicitly specify that a pod being created as a Kata Containers pod. For details, please refer to [How to use Kata Containers and Containerd](../../docs/how-to/containerd-kata.md).
+With `RuntimeClass`, users can define Kata Containers as a
```

Comment:
Does RuntimeClass work with CRI-O?  I would try to more clearly separate out how crio and containerd work in this section. They seem to be quite different.

Reply:
They should be the same, Eric!

Reply:
What I was trying to say is that containerd is mentioned in this section but no mention of CRIO.  There are other parts of this guide where only containerd is mentioned leading the reader to think this particular section is only containerd. I think mentioning CRIO here would help a lot.